incrementalbuild/src/test/java/io/takari/incrementalbuild/spi/DefaultAggregatorBuildContextTest.java

```diff
@@ -133,4 +133,18 @@ public void testEmpty() throws Exception {
     Assert.assertEquals(1, indexer.outputs.size());
   }
 
+  @Test
+  public void testEmptyNoCreate() throws Exception {
```

Comment:
Please add a test for incremental delete of an aggregate output created during a prior build.